src/core/src/pass/serialize.cpp

```diff
@@ -904,9 +905,6 @@ void serialize_rt_info(pugi::xml_node& root, const std::string& name, const ov::
     }
 }
 
-// TODO: Make this definitiion unuque, now it is duplicated between this file and compress_float_constants.cpp
-const std::string& postponed_fp16_compression_tag = "postponed_fp16_compression";
```

Comment:
Move it to [src/common/transformations/src/transformations/rt_info/disable_fp16_compression.cpp](https://github.com/slyalin/openvino/pull/125/files#diff-014eb2acd10b982b82884f46dacd832f0794bc7f449ccba2b2f0ff324a5e0ac7) where it will be used in set/get methods.